tests/kernels/test_cutlass_moe.py

```diff
@@ -131,9 +131,9 @@ def test_cutlass_moe_no_graph(
                                          c_strides2,
                                          a1_scale=a_scale1)
 
-        print(triton_output)
-        print(cutlass_output)
-        print("*")
+        #print(triton_output)
```

Comment:
uncomment?

Reply:
These prints were just spamming the test output with tensors.